### PR TITLE
Clearing in DHCP

### PR DESCRIPTION
**Pull request by joabda**
Fixing error messages in existing clear.
Adding new clear for DHCP Snooping statistics.

### CODE REVIEW

File: pkgs/sdk-pkg/src/genie/libs/sdk/apis/iosxe/dhcp/execute.py [hunk 1]
```diff
@@ -39,7 +39,24 @@ def clear_ip_dhcp_snooping_binding(device):
...
+def clear_ip_dhcp_snooping_statistics(device):
+    """execute 'clear ip dhcp snooping statistics *' on device
+       Args:
+            device('obj'): device object
+       Returns:
+            None
+       Raises:
+            SubCommandFailure
+    """
+    log.info("Executing clear_ip_dhcp_snooping_statistics API")
```
Comment (ThomasJRyan) [inline pkgs/sdk-pkg/src/genie/libs/sdk/apis/iosxe/dhcp/execute.py:54]:
```suggestion
    log.debug("Executing clear_ip_dhcp_snooping_statistics API")

```

**Review by ThomasJRyan — CHANGES_REQUESTED** [review]
Thank you for this new API. We'll need a changelog and unittest to get this merged in!

**Comment by joabda** [comment]
> Thank you for this new API. We'll need a changelog and unittest to get this merged in!

Could you explain to me how to perform these please? First time adding new CLI.

**Comment by ThomasJRyan** [comment]
Closing due to internal matters

**Pull request closed without merging**